tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/RabbitMQ/ContextPropagation.cs

```diff
@@ -12,12 +12,12 @@ namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.RabbitMQ
 {
     internal static class ContextPropagation
     {
-        public static void HeadersSetter(IDictionary<string, object> carrier, string key, string value)
+        public static readonly Action<IDictionary<string, object>, string, string> HeadersSetter = (carrier, key, value) =>
         {
             carrier[key] = Encoding.UTF8.GetBytes(value);
-        }
+        };
 
-        public static IEnumerable<string> HeadersGetter(IDictionary<string, object> carrier, string key)
+        public static readonly Func<IDictionary<string, object>, string, IEnumerable<string>> HeadersGetter = (carrier, key) =>
```

Comment:
```suggestion
        public static readonly Func<IDictionary<string, object>, string, IEnumerable<string>> HeadersGetter = static (carrier, key) =>
```

Reply:
Added in 31aee89. I cannot see any changes in the codegen as expected.